pkg/lib/ldap/entry.go

```diff
@@ -2,6 +2,21 @@ package ldap
 
 import "github.com/go-ldap/ldap/v3"
 
+const (
+	AttributeNameEmail    = "mail"
+	AttributeNameMobile   = "mobile"
+	AttributeNameUsername = "uid"
```

Comment:
Any references on these attributes? Are they well-known? Please include the references as comments.